packages/governance/src/contractHelper.js

```diff
@@ -4,14 +4,30 @@ import { Far } from '@endo/marshal';
 import { makeStoredPublisherKit } from '@agoric/notifier';
 import { getMethodNames, objectMap } from '@agoric/internal';
 import { ignoreContext } from '@agoric/vat-data';
-import { keyEQ } from '@agoric/store';
+import { keyEQ, M } from '@agoric/store';
+import { AmountShape, BrandShape } from '@agoric/ertp';
 import { assertElectorateMatches } from './contractGovernance/paramManager.js';
 import { makeParamManagerFromTerms } from './contractGovernance/typedParamManager.js';
 
 const { details: X, quote: q } = assert;
 
 export const GOVERNANCE_STORAGE_KEY = 'governance';
 
+const extendPublicAPI = {
```

Comment:
```suggestion
const extendPublicAPI = harden({
```

Reply:
It's currently only provided in a hardened object, so not actually necessary. However it should move to an exported constant so aye.